.gitignore

```diff
@@ -1,5 +1,5 @@
 .env
-/react-uswds-sample
+/your-path
```

Comment:
Wait actually would it be better to just keep both our own paths in the .gitignore and tell them to add in their path instead of replacing it? That way we don't need to remember to add back our own path every time we push & pull from Github

Reply:
Yeah that works. Would you prefer that I add in our paths now or we both just remember to do it the next time we create a branch and then we will keep it in from then on?